pkgs/applications/altcoins/mist-browser.nix

```diff
@@ -0,0 +1,81 @@
+{ pkgs, stdenv, alsaLib, atk, cairo, cups, dbus, dpkg, expat, fetchurl,
+fontconfig, freetype, gdk_pixbuf, gnome, glib, gtk, go-ethereum, libcap,
```

Comment:
Please take a look at the coding style guidelines at https://nixos.org/nixpkgs/manual/#chap-conventions